interface/resources/qml/+webengine/QmlWebWindowView.qml

```diff
@@ -40,7 +40,7 @@ Controls.WebView {
     userScripts: [ createGlobalEventBridge, raiseAndLowerKeyboard, userScript ]
 
     function onWebEventReceived(event) {
-        if (event.slice(0, 17) === "CLARA.IO DOWNLOAD") {
+        if (typeof event === 'string' && event.slice(0, 17) === "CLARA.IO DOWNLOAD") {
```

Comment:
Use "s instead of 's for string delimiters, to match the rest of the code in the file?
Ditto for other changes.